sparkbench/ml/src/main/scala/com/intel/sparkbench/ml/LinearRegressionWithElasticNet.scala

```diff
@@ -0,0 +1,119 @@
+package com.intel.hibench.sparkbench.ml
+
+import org.apache.spark.ml.feature.LabeledPoint
+import org.apache.spark.ml.linalg.Vector
+import org.apache.spark.ml.regression.LinearRegression
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import scopt.OptionParser
+
+object LinearRegressionWithElasticNet {
+  case class Params(
+      input: String = null,
+      regParam: Double = 0.3,
+      elasticNetParam: Double = 0.8,
+      maxIter: Int = 10,
+      tol: Double = 1E-6,
+      fracTest: Double = 0.25)
```

Comment:
These parameters should be added to linear.conf?

Reply:
@carsonwang  Yes , I discussed this with xiaochang, he will submit PR and put the parameters that need to be configured into the linear.conf.